docs/post_installation/post_installation.md

```diff
@@ -0,0 +1,166 @@
+
+Depending on installation path you selected, default config files may be also a part of the installation experience and so you may see the following configuration files at your deployment directory:
+```bash
+config.json
+config_chrysalis_testnet.json
+config_comnet.json
+config_devnet.json
+peering.json
+profiles.json
+```
+
+### Default configuration
+By default, Hornet searches for configuration files in a working directory and expects default names, such as `config.json` and `peering.json`.
+
+This behavior can be changed by running Hornet with some altering arguments. The default directory with configuration files can be changed by running `hornet --config-dir` argument.
+
+Please see [config.json](./config.md) and [peering.json](./peering.md) chapters for more information regarding respective configuration files.
+
+Once Hornet is executed, it outputs all loaded configuration parameters to `stdout` to be sure what configuration was loaded.
+
+All other altering command line parameters can be obtained by running `hornet --help`.
+
+> Hornet version 0.5.x targets IOTA 1.0 mainnet network by default. Hornet version 0.6.x targets IOTA 1.5 (Chrysalis) mainnet network by default
+
+### Identifying a configuration file based on particular use cases
+There is a simple way how to recognize which configuration file targets which IOTA network:
+```bash
+cat config.json | jq "[.httpAPI?, .restAPI?]"
+```
+* `jq` command line json parser can be installed using `sudo apt install jq`
+
+IOTA 1.5 (Chrysalis network) provides the following rest API endpoints:
+```json
+[
+  null,
+  {
+    "jwtAuth": {
+      "enabled": false,
+      "salt": "HORNET"
+    },
+    "excludeHealthCheckFromAuth": false,
+    "permittedRoutes": [
+      "/health",
+      "/mqtt",
+      "/api/v1/info",
+      "/api/v1/tips",
+      "/api/v1/messages/:messageID",
+      "/api/v1/messages/:messageID/metadata",
+      "/api/v1/messages/:messageID/raw",
+      "/api/v1/messages/:messageID/children",
+      "/api/v1/messages",
+      "/api/v1/transactions/:transactionID/included-message",
+      "/api/v1/milestones/:milestoneIndex",
+      "/api/v1/milestones/:milestoneIndex/utxo-changes",
+      "/api/v1/outputs/:outputID",
+      "/api/v1/addresses/:address",
+      "/api/v1/addresses/:address/outputs",
+      "/api/v1/addresses/ed25519/:address",
+      "/api/v1/addresses/ed25519/:address/outputs",
+      "/api/v1/treasury"
+    ],
+    "whitelistedAddresses": [
+      "127.0.0.1",
+      "::1"
+    ],
+    "bindAddress": "0.0.0.0:14265",
+    "powEnabled": true,
+    "powWorkerCount": 1,
+    "limits": {
+      "bodyLength": "1M",
+      "maxResults": 1000
+    }
+  }
+]
+```
+
+IOTA 1.0 provides legacy rest API endpoints:
```

Comment:
No point in including instructions for IOTA 1.0 here.

Reply:
Removed